miscellaneous/assessments/week_1_assessment.md

```diff
@@ -0,0 +1,21 @@
+##Question 1 - Variables
+Create a variable and set it equal to 'variable'.  
+What are some different data types? Write a few examples.
+
+##Question 2 - Hashes and Arrays
+Create a new folder called 'cars' and then a file called 'my_cars.rb'  
+Add two hashes, one for each `car`, with the following attributes: `wheels`, `max_speed`, `color`  
+Create an array that contains both cars.  
+How do we use the array to access the second car? How do we find the second car's `color`?
+
+## Question 2 - Classes and Methods
+Create a new file called 'car.rb' with:
+ - a class Car
+ - a method that "paints" a car a new color.
+ - a method that checks if the current car is the first car (`is_first_car?`)
```

Comment:
Sorry for late comment on this but @AmberWilkie, how do you expect them to check for this?